pipeline/lib/storage.ml

```diff
@@ -108,7 +112,10 @@ let record_success ~serial_id (db : Postgresql.connection) =
   let query =
     Fmt.str
       {|UPDATE benchmark_metadata
-        SET success = true
+        SET success = true,
+            failed = false,
+            cancelled = false,
+            reason = ''
```

Comment:
I don't think it really matters, but I guess `reason` was supposed to be `NULL` when unset